distribution/mpg.properties

```diff
@@ -7,4 +7,14 @@ email = firstName.lastName@gmail.com
 password = foobar
 
 # Enable auto-update team ('false' by default)
-#team.update=true
\ No newline at end of file
+#team.update=true
+
+# Usage behind a company proxy
+#proxy.uri = http://company.proxy.com:80
+#proxy.user = foo
+#proxy.password = bar
+
+# Notes for tactical substitutes
+tactical.substitute.attacker = 4.5
```

Comment:
The default `mpg.properties` given in distribution should contains the default values, so perhaps 
```
#tactical.substitute.attacker = 6.0
#tactical.substitute.midfielder = 5.0
#tactical.substitute.defender = 5.0
```

WDYT ?

Reply:
I Agree